docs/sources/flow/reference/components/otelcol.exporter.otlphttp.md

```diff
@@ -0,0 +1,205 @@
+refers to a `tls` block defined inside a `client` block.
+
+[client]: #client-block
+[tls]: #tls-block
+[queue]: #queue-block
+[retry]: #retry-block
+
+### client block
+
+The `client` block configures the HTTP client used by the component.
+
+The following arguments are supported:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`endpoint`           | `string`      | `host:port` to send telemetry data to. | | yes
+`read_buffer_size`   | `string`      | Size of the read buffer the HTTP client uses for reading server responses. | `0` | no
+`write_buffer_size`  | `string`      | Size of the write buffer the HTTP client uses for writing requests. | `"512KiB"` | no
+`timeout`            | `duration`    | Time to wait before marking a request as failed. | `"30s"` | no
+`headers`            | `map(string)` | Additional headers to send with the request. | `{}` | no
+`compression`        | `string`      | Compression mechanism to use for requests. | `"gzip"` | no
+`max_idle_conns`     | `int`         | Limits the number of idle HTTP connections the client can keep open. | `100` | no
+`max_idle_conns_per_host` | `int`    | Limits the number of idle HTTP connections the host can keep open. | `0` | no
+`max_conns_per_host` | `int`         | Limits the total (dialing,active, and idle) number of connections per host. | `0` | no
+`idle_conn_timeout`  | `duration`    | Time to wait before an idle connection will close itself. | `"90s"` | no
+`auth`               | `capsule(otelcol.Handler)` | Handler from an `otelcol.auth` component to use for authenticating requests. | | no
+
+By default, requests are compressed with gzip. The `compression` argument
+controls which compression mechanism to use. Supported strings are:
+
+* `"gzip"`
+* `"zlib"`
+* `"deflate"`
+* `"snappy"`
+* `"zstd"`
+
+If `compression` is set to `"none"` or an empty string `""`, no compression is
+used.
+
+### tls block
+
+The `tls` block configures TLS settings used for the connection to the HTTP
+server.
+
+The following arguments are supported:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`ca_file` | `string` | Path to the CA file. | | no
+`cert_file` | `string` | Path to the TLS certificate. | | no
+`key_file` | `string` | Path to the TLS certificate key. | | no
+`min_version` | `string` | Minimum acceptable TLS version for connections. | `"TLS 1.2"` | no
+`max_version` | `string` | Maximum acceptable TLS version for connections. | `"TLS 1.3"` | no
+`insecure` | `boolean` | Disables TLS when connecting to the HTTP server. | | no
+`insecure_skip_verify` | `boolean` | Ignores insecure server TLS certificates. | | no
+`server_name` | `string` | Verifies the hostname of server certificates when set. | | no
+
+The `tls` block should always be provided, even if the server doesn't support
+TLS. To disable `tls` for connections to the HTTP server, set the `insecure`
```

Comment:
@rfratto I copy/pasted this from the `otlp` exporter. We say that this block should always be provided, but it's not a required one. Should we relax this doc sentence (eg. s/always be/always can/)?

Reply:
Yeah, this is my fault, I think I misspoke originally. 

I think it's probably more accurate to say:

> If the server doesn't support TLS, the `tls` block must be provided with the `insecure` argument set to `true`. 

and drop the recommendation about it being provided, since I don't think you _need_ it for connecting to TLS servers unless the server does client certificate auth.

WDYT?